test/e2e/storage/testsuites/volumes.go

```diff
@@ -139,8 +149,11 @@ func (t *volumesTestSuite) defineTests(driver TestDriver, pattern testpatterns.T
 		validateMigrationVolumeOpCounts(f.ClientSet, dInfo.InTreePluginName, l.intreeOps, l.migratedOps)
 	}
 
-	ginkgo.It("should be mountable", func() {
-		skipPersistenceTest(driver)
```

Comment:
`skipPersistenceTest` becomes unused now? Then, it would be better to delete it.

Reply:
Good catch, removed.